### ISSUE TITLE
Input Labels Issues

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1680]
The following filter inputs for tables don't have their corresponding labels associated with them and are missing `aria-label` and `aria-roles` attributes:
- Call filter
- Technique filter
- Experimental area filter
- Status filter
